docs/Makefile

```
@@ -0,0 +1,31 @@
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXAUTOBUILD = sphinx-autobuild
+SPHINXPROJ    = Leapp repository
+SOURCEDIR     = source
+BUILDDIR      = build
+
+all: clean html
+
+help:
+	@source venv/bin/activate && \
+	$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+livehtml:
```

```suggestion
livehtml: html
```

this will automatically build html without the need to think about it